config/routes.rb

```diff
@@ -41,4 +41,6 @@
 
   resources :bids, only: [:create, :destroy]
   get "/dashboard/bids", to: "bids#index", as: :user_bids
+
+  post '/notification' => 'notification#create'
```

Comment:
This is kind of nitpicky but just to keep consistency we should do `post "/notification", to: "notification#create"`